### PR TITLE
Done with image issue and created constant folder

### PR DESCRIPTION
**Pull request by samrudhi-Gamhire**

### CODE REVIEW

File: src/app/gallery-page-component/gallery-page-component.ts [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
-
-export class MyGalleryPageComponent  {
-
-    
-}   
\ No newline at end of file
+    export class MyGalleryPageComponent  {
+        
+        gallery = []; 
+        labels =["all","indoor","outdoor","gd","babies","monthlyactivty","specialevent","birhday","schoolreadlines"];
```
Comment (amit-gaikwad) [inline src/app/gallery-page-component/gallery-page-component.ts:13]: Labels should come from constant files
incase if we want to change it then we can see the constant file only

File: src/app/services/gallery-service.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+constructor(private http:Http){}
+ link=GlobalUrl.url;
+getAll() : Observable <any> {
+return this.http.get(this.link).
+pipe(map((res:Response)=>{
+    return res.json();
+}));
+}
+
+getByCategary(categary : string) : Observable <any>{
```
Comment (amit-gaikwad) [inline src/app/services/gallery-service.ts:19]: use proper indentation for code. to better understand next time

**Review by amit-gaikwad — CHANGES_REQUESTED** [review]
Resolve comments

File: src/app/services/gallery-service.ts (hunk 2 above)
Comment (samrudhi-Gamhire) [reply src/app/services/gallery-service.ts:19]: should i add commends for better identation of code

**Review by amit-gaikwad — APPROVED** [review]

**Pull request merged**